app/services/file_upload/file_uploader.rb

```diff
@@ -0,0 +1,8 @@
+module FileUpload
+  class FileUploader
+    def destroy(file_path)
```

Comment:
Do you not also want to consider file existence in the bucket? I don't know if trying to delete a missing file throws an exception and I don't think it should stop the whole job running.